rpc/wrtc_base_channel.go

```diff
@@ -144,9 +147,20 @@ func newBaseChannel(
 func (ch *webrtcBaseChannel) Close() {
 	// RSDK-8941: Having this instead early return when `closed` is set will result in `TestServer`
 	// to leak goroutines created by `dialWebRTC`.
```

Comment:
Is this comment relevant anymore? Seems like we early return now, but we do wait for `activeBackgroundWorkers`. Is that enough to stop the leaked goroutines?